src/probe_modules/module_bacnet.c

```diff
@@ -73,7 +73,8 @@ int bacnet_init_perthread(void *buf, macaddr_t *src,
 }
 
 int bacnet_make_packet(void *buf, ipaddr_n_t src_ip, ipaddr_n_t dst_ip,
-				uint32_t *validation, int probe_num, UNUSED void *arg)
+				uint32_t *validation, int probe_num, UNUSED void *arg,
+				UNUSED int *len)
```

Comment:
Placing it next to `buf` seemed "disruptive" given that it's essentially never used. I followed the lead from `arg` which is also never used, and placed it at the end.

But I'm not opposed. Do you want me to make the change?